client/js/main.ts

```diff
@@ -143,55 +144,21 @@ function checkIn (e: Event) {
 	  }
 	}`;
 
-	qwest.post("/graphql", JSON.stringify({
-		query: mutation,
+	client.mutate<GQL.IMutation>({
+		mutation: mutation,
 		variables: {
 			user: id,
 			tag: tag
 		}
-	}), graphqlOptions).catch((e, xhr, response) => {
-		alert(response.error);
-	}).complete(() => {
+	}).then(response => {
+		button.disabled = false;
+	}).catch(error => {
+		console.error(error);
+		alert("Error checking in participant");
 		button.disabled = false;
 	});
 }
 
-function attachUserDeleteHandlers () {
```

Comment:
Why is this account deletion functionality being removed?

Reply:
Honestly, I think I just did this by accident while moving things around. Thanks for pointing this out!